lib/Utils/Logging.pm

```diff
@@ -324,7 +324,6 @@ sub export_logs_basic {
     save_and_upload_log('journalctl -b -o short-precise', '/tmp/journal.txt', {screenshot => 1});
     save_and_upload_log('dmesg', '/tmp/dmesg.txt', {screenshot => 1});
     tar_and_upload_log('/etc/sysconfig', '/tmp/sysconfig.tar.gz', {gzip => 1});
-    record_avc_selinux_alerts();
```

Comment:
with this line removed, now opensusebasetest is also missing a call to `record_avc_selinux_alerts`.

Reply:
Yes, that is because kernel tests are using this library and those do not really need to detect AVC. In case a test needs to call this function, it should call it in its module hooks